core/App/ehsm_core_test.cpp

```diff
@@ -41,6 +41,9 @@
 #include "dsohandle.h"
 #include "json_utils.h"
 
```

Comment:
commit tiltle:
"
Dynamic parse the mr_enclave and mr_signer in the unittest

passed the quote verification test
"